src/include/ipxe/efi/LoongArch64/ProcessorBind.h

```diff
@@ -30,17 +30,17 @@ FILE_LICENCE ( BSD2_PATENT );
 // Assume standard LoongArch 64-bit alignment.
 // Need to check portability of long long
 //
-typedef unsigned long   UINT64;
-typedef long            INT64;
-typedef unsigned int    UINT32;
-typedef int             INT32;
-typedef unsigned short  UINT16;
-typedef unsigned short  CHAR16;
-typedef short           INT16;
-typedef unsigned char   BOOLEAN;
-typedef unsigned char   UINT8;
-typedef char            CHAR8;
-typedef char            INT8;
+typedef unsigned long long UINT64;
+typedef long long          INT64;
+typedef unsigned int       UINT32;
+typedef int                INT32;
+typedef unsigned short     UINT16;
+typedef unsigned short     CHAR16;
+typedef short              INT16;
+typedef unsigned char      BOOLEAN;
+typedef unsigned char      UINT8;
+typedef char               CHAR8;
+typedef char               INT8;
```

Comment:
This looks sensible, but will need to be pushed to https://github.com/tianocore/edk2 first since that is the upstream source for this imported header.

Reply:
Thanks, changes to edk2 are in progress.

Reply:
https://bugzilla.tianocore.org/show_bug.cgi?id=4330

Reply:
Thanks - that was a quick turnaround!  Imported to iPXE just now in commit https://github.com/ipxe/ipxe/commit/aa85c2918